### PR TITLE
Sysbench Io Benchmarking 

### LINKED ISSUE #16: [Chaos] Add support to fileio test
## Please describe what you would like to see in this project

Add support to fileio options in the input schema, add related commandline arguments to the plugin, parse the output of the fileio test.

## Please describe your use case

Create disk io hog scenario using arcaflow

### PR DESCRIPTION
**Pull request by tsebastiani**
parsing function and schema update for Random output

added unit tests

unit tests

## Changes introduced with this PR

Enabling the sysbench io benchmark functionality for chaos purpose.

---
By contributing to this repository, I agree to the [contribution guidelines](https://github.com/arcalot/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: arcaflow_plugin_sysbench/sysbench_schema.py [hunk 1]
```diff
@@ -223,6 +249,116 @@ class SysbenchMemoryInputParams(CommonInputParameters):
     ] = SeqRnd.SEQ
 
 
+@dataclass
+class SysbenchIoInputParams(CommonInputParameters):
+    """
+    This is the data structure for the
+    input parameters of Sysbench CPU benchmark.
```
Comment (dustinblack) [inline arcaflow_plugin_sysbench/sysbench_schema.py]: Extremely minor nit -- Should be "IO benchmark" not "CPU benchmark"

File: arcaflow_plugin_sysbench/sysbench_schema.py [hunk 2]
```diff
@@ -433,6 +626,37 @@ class SysbenchCpuResultParams:
...
+    ]
+    Throughput: typing.Annotated[
+        ThroughputMetrics,
+        schema.name("Throughput"),
+        schema.description("Throughput metrics"),
+    ]
+    Latency: typing.Annotated[
+        LatencyAggregates,
+        schema.name("Latency"),
+        schema.description("CPU latency in miliseconds"),
```
Comment (dustinblack) [inline arcaflow_plugin_sysbench/sysbench_schema.py]: Is this actually CPU latency? Or file operation latency?

**Review by dustinblack — CHANGES_REQUESTED** [review]
Very minor nits

**Review by Harshith-umesh — COMMENTED** [review]
@tsebastiani could you please add an example config for the io tests here: https://github.com/arcalot/arcaflow-plugin-sysbench/tree/io_benchmark/configs

File: arcaflow_plugin_sysbench/sysbench_schema.py (hunk 2 above)
Comment (tsebastiani) [reply arcaflow_plugin_sysbench/sysbench_schema.py]: I've no clue about that, what I know is that the output structure is the same as for the other tests, so I supposed it was the same kind of "Latency".

File: arcaflow_plugin_sysbench/sysbench_schema.py (hunk 1 above)
Comment (tsebastiani) [reply arcaflow_plugin_sysbench/sysbench_schema.py]: fixed and pushed.

**Comment by tsebastiani** [comment]
> @tsebastiani could you please add an example config for the io tests here: https://github.com/arcalot/arcaflow-plugin-sysbench/tree/io_benchmark/configs

added basic configuration example

File: arcaflow_plugin_sysbench/sysbench_schema.py (hunk 2 above)
Comment (dustinblack) [reply arcaflow_plugin_sysbench/sysbench_schema.py]: It's hard to find an absolutely clear answer, and my C skills are pretty atrophied, but I feel very sure that this latency value is for the file operation, not the CPU.

**Review by Harshith-umesh — APPROVED** [review]
LGTM!

**Review by dustinblack — APPROVED** [review]

**Pull request merged**